web-client/src/presenter/actions/CaseDetail/editRemoteStatusAction.ts

```diff
@@ -0,0 +1,41 @@
+import { state } from '@web-client/presenter/app.cerebral';
+import { Case } from '@shared/business/entities/cases/Case';
+
+/**
+ * calls the editRemoteStatusInteractor to indicate remote status
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {Function} providers.get the cerebral get function
+ * @returns {object} the alertSuccess and updated caseDetail object
+ */
+export const editRemoteStatusAction = async ({
+  applicationContext,
+  get,
+}: ActionProps) => {
+  const docketNumber = get(state.caseDetail.docketNumber);
+  const modalState = get(state.modal);
+  const { remoteTrialGrantedDate } = modalState;
+  const rawCase = get(state.caseDetail);
+  const user = get(state.user);
+  const currentCase = new Case(rawCase, { 
+    authorizedUser: user,
+    filtered: false,
+    isNewCase: false 
+  });
+  
+  currentCase.setRemoteTrialGrantedDate(remoteTrialGrantedDate);
```

Comment:
The filtered and isNewCase parameters on the case both default to false, so we could omit them and make this a one line call. 

```suggestion
  const user = get(state.user);
  const currentCase = new Case(rawCase, { authorizedUser: user });

  currentCase.setRemoteTrialGrantedDate(remoteTrialGrantedDate);
```

Reply:
I like the suggestion. It improves readability.